proto/meta.proto

```diff
@@ -399,3 +400,17 @@ service SystemParamsService {
   rpc GetSystemParams(GetSystemParamsRequest) returns (GetSystemParamsResponse);
   rpc SetSystemParam(SetSystemParamRequest) returns (SetSystemParamResponse);
 }
+
+message RwCloudValidateSourceRequest {
+  connector_service.SourceType source_type = 1;
```

Comment:
I think it is better to define a new enum inside `RwCloudValidateSourceRequest`, and put the interface to a dedicated proto file.